lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -109,19 +99,25 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
 
     /** True if tags should be displayed, false otherwise */
     @Input()
-    displayTags = false;
+    displayTags = true;
```

Comment:
Why did we change default for displayTags? Is not that breaking change for projects which use ADF/external customer as they will need update their code to use false now?

Reply:
in case you will leave that change, please update documentation too

Reply:
Hi, @DenysVuika Can you please provide more context/reason of changing this default value of displayTags?

Reply:
reverted to original defaults